person.rb

```diff
@@ -1,16 +1,17 @@
-require 'SecureRandom'
 require './nameable'
 
 class Person < Nameable
   attr_accessor :name, :age
   attr_writer :id
+  attr_reader :rental
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     super()
-    @id = SecureRandom.uuid
+    @id = Random.rand(1..1000)
     @age = age
     @name = name
     @parent_permission = parent_permission
+    @rental = []
   end
 
```

Comment:
- [OPTIONAL] To implement the `has-many` side of the association between `person` and `rental`, `a method for adding rentals should be present` in the `Person class`. Something like so;
```ruby
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
```
The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association, please kindly fix  that.